specs/insights/paths/pushEvents.yml

```
@@ -2,105 +2,262 @@ post:
+                This can happen if you're sending an empty `events` array,
+                or if you try to send a single event object instead of the `events` array.
+              value: {'status': 422, 'message': 'No events to process'}
+            NoEventsField:
+              summary: Missing events attribute.
+              description: This can happen if you try to send an array of events directly instead of using the `events` attribute.
+              value: {'status': 422, 'message': 'Invalid type for field : expected insights.PublicEventsBatch, got array'}
+            EventNameRequired:
+              summary: Missing eventName attribute.
+              value: {'status': 422, 'message': 'EventName is required'}
+            WrongEventName:
+              summary: Event name too long or wrong characters.
+              value: {'status': 422, 'message': 'EventName must contain only visible ascii characters, and be between 1 and 64 characeters long'}
+            EventTypeRequired:
+              summary: Missing eventType attribute.
+              value: {'status': 422, 'message': 'EventType is required'}
+            WrongEventType:
+              summary: Wrong event type.
+              value: {'status': 422, 'message': 'EventType must be one of "click", "conversion" or "view"'}
+            IndexRequired:
+              summary: Missing index attribute.
+              value: {'status': 422, 'message': 'The index field is required'}
+            UserTokenRequired:
+              summary: Missing userToken attribute.
+              value: {'status': 422, 'message': 'The userToken field is required'}
+            WrongUserToken:
+              summary: Malformed user token.
+              value: {'status': 422, 'message': 'UserToken must contain only alphanumeric, equal, plus, slash, hyphen, or underscore characters, and be between 1 and 128 characters long'}
+            NoObjectIDsOrFilters:
+              summary: Missing objectIDs or filters attribute.
+              description: Each event must include either the `objectIDs` or `filters` attribute.
+              value: {'status': 422, 'message': 'Event should specify either some ObjectIDs or some Filters'}
+            BothDefined:
+              summary: Both objectIDs and filters attributes defined.
+              value: {'status': 422, 'message': 'Event should specify ObjectIDs or Filters, but not both'}
+            PositionsOnWrongEventType:
+              summary: Wrong combination of positions and eventType attributes.
+              value: {'status': 422, 'message': 'Only event of type click should specify the positions attribute'}
+            PositionsWithoutQueryID:
+              summary: Positions attribute without queryID.
+              description: Click events with the `positions` attribute require the `queryID` attribute.
+              value: {'status': 422, 'message': 'Event of type click with positions should specify a queryID'}
```

```suggestion
              value: {'status': 422, 'message': 'Event of type "click with positions" should specify a queryID'}
```
"Click with positions" or "click"?